src/api/dependencies.py

```
@@ -0,0 +1,631 @@
+                """Process files using real batch processor."""
+                try:
+                    # Create batch processing job
+                    job = await self.processor.create_batch_job(
+                        file_paths=file_paths,
+                        job_id=job_id
+                    )
+                    
+                    # Process the batch
+                    result = await self.processor.process_batch(job)
+                    
+                    return result
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)